cmd/api/src/database/file_upload.go

```diff
@@ -64,6 +64,10 @@ func (s *BloodhoundDB) GetAllFileUploadJobs(skip int, limit int, order string, f
 		result = s.db.Model(model.FileUploadJob{}).Count(&count)
 	}
 
+	if result.Error != nil {
+		return nil, 0, CheckError(result)
+	}
```

Comment:
This is to fix an issue the linter was warning me about. We weren't doing anything with the result value after doing a count, so I added error checking after the count step.